.gitignore

```diff
@@ -129,6 +129,3 @@ ENV/
 
 # We want this to only be created on CI/CD platforms
 VERSION
-
-# Since we use a Jinja Template, we only want this to be created as build time
-meta.yaml
```

Comment:
We can prob keep this here incase we ever go back to trying to automate this.

Reply:
This was removed since this branch started as actually trying to commit `meta.yaml`. You're saying I should leave it in in case we go back to templating (à la #300) again?

Reply:
Either way 🤷